backend/users/api/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import RegisterAPI
+
+urlpatterns = [
+    path('register/', RegisterAPI.as_view(), name='register')
+]
```

Comment:
add new line always at end of file, pls also apply this to other files that has like this icon ⛔